test/selection_pile_validator.test.ts

```diff
@@ -176,4 +177,20 @@ describe("validate", () => {
       expect(validator.validate(selectionPile)[0].message).to.equal("exceeded_list_limit")
     });
   })
+
+  context("with lazy errors", ()=> {
+    context("with below list limit", () => {
```

Comment:
This doesn't appear to test lazyness?

Reply:
It was just to have all lazy errors tests under a scope.

Reply:
below list limit is a lazy error